### ISSUE TITLE
MCA Villager/Entity Rendering Issues

### ISSUE THREAD

**Issue by Mewcenary** [Luke100000/minecraft-comes-alive#213]
Java: 8 - 331
Forge:  1.16.5-forge-36.2.35 etc.
minecraft-comes-alive-7.0.11-forge

[crash-2022-05-12_01.32.35-client.txt](https://github.com/Luke100000/minecraft-comes-alive/files/8677320/crash-2022-05-12_01.32.35-client.txt)

**Comment by Luke100000** [comment]
Seems like a mod incompatibility, when does it crash? Can you reproduce it?. It might be in relation to the 2 players on a horse mod, but for me it works. Cavesandcliffs also modifies a lot.

**Comment by Mewcenary** [comment]
When I was in creative mode doing some building, flying, using spawn eggs(iron golem and guard villagers), testing the pillager raids, or doing nothing just standing there.  

I had to go back to all the older mode updates for It to fix.  When I removed the horse and backdated Cavesandcliffs, it still had rendering issues.  

Funny enough, its-meow the creator of BetterAnimalsPlus, also recommended the same thing, and no dice.  Not sure if you want to close this issue, or monitor it, because I changed everything to older mods for it to work.  Not even my Apex Hosting Server techs could figure out what was wrong with it.
